package.json

```diff
@@ -96,7 +96,9 @@
     "mkdirp": "^0.5.1",
     "phantomjs": "^1.9.18",
     "raw-loader": "^0.5.1",
-    "react": "^0.13.3",
+    "react": "^0.14.0",
+    "react-addons-test-utils": "^0.14.0",
```

Comment:
I assume this needs to go into actual dependencies so that other Core projects get it automatically for their tests?